docs/install/visual-studio-on-arm-devices.md

```diff
@@ -66,9 +66,9 @@ With the Visual Studio 17.3 Preview 2 release, you can access the new native Arm
 
 | **Host architecture (platform the compiler is running on)** | **Target architecture (platform the compiler is generating binaries for)** | **Installation path** |
 | ----------- | ------ | --------------- |
-| Arm64 | Arm64 | <'Visual Studio install'>\VC\Tools\MSVC\14.33.31424\bin\HostARM64\ARM64 |
-| Arm64 | X64 | <'Visual Studio install'>\VC\Tools\MSVC\14.33.31424\bin\HostARM64\x64 |
-| Arm64 | X86 | <'Visual Studio install'>\VC\Tools\MSVC\14.33.31424\bin\HostARM64\x86 |
+| Arm64 | Arm64 | <Install location>\VC\Tools\MSVC\<version>\bin\HostARM64\ARM64 |
```

Comment:
These lines are giving docs build validation errors, since the build is interpreting them as HTML.  I think they have to be escaped with a backslash.
```suggestion
| Arm64 | Arm64 | \<Install location\>\VC\Tools\MSVC\<version>\bin\HostARM64\ARM64 |
```